cmd/release/cmd/update.go

```diff
@@ -130,11 +132,44 @@ var updateChartsCmd = &cobra.Command{
 	},
 }
 
+var updateRancherCmd = &cobra.Command{
+	Use:   "rancher",
+	Short: "Update rancher files",
+}
+
+var updateRancherDashboardCmd = &cobra.Command{
+	Use:   "dashboard [version]",
+	Short: "Update Rancher's Dashboard and UI references and create a PR",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return errors.New("expected at least one argument: [version]")
+		}
+
+		version := args[0]
```

Comment:
Can we get some validation on this value to make sure a correct value was given?

Reply:
Added the `semver` validation.